src/Time/Units.hs

```diff
@@ -266,6 +267,10 @@ fortnight :: Natural -> Fortnight
 fortnight = time
 {-# INLINE fortnight #-}
 
+-- | Similar to 'floor', but works with 'Time' units.
```

Comment:
I would like to have several examples in Haddock how this function works. These examples also can be put into unit tests.